test.py

```diff
@@ -53,9 +53,6 @@ def gen_test_params():
             continue
         # Test encoding detection for each file we have of encoding for
         for file_name in listdir(path):
-            ext = splitext(file_name)[1].lower()
-            if ext not in ['.html', '.txt', '.xml', '.srt']:
```

Comment:
The main reason for it is to deal with hidden files that can crop up on certain OSes.